scripts/generateNewContributors.js

```diff
@@ -9,9 +9,7 @@ twitter: ""
 linkedin: ""
 slack: ""
 joining_date: ""
-core: false
-intern: false
-operations: false
+role : "contributor"
```

Comment:
```suggestion
role : contributor
```

Reply:
should i change script as well ? @rithviknishad